JS_HW_4/task2.js

```diff
@@ -0,0 +1,70 @@
+// 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив)
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+//   Пиццы конкурента:
+//   const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+
+const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+
+const filterPizza = pizzaNames => {
+  const filteredNames = pizzaNames.filter(
+    name => !competitorPizzas.map(competitorPizza => competitorPizza.toLowerCase()).includes(name.toLowerCase())
```

Comment:
Better to split this actions, because you perform map for each interaction of filter. You should map before filter in separate row